packages/client/src/registry/OperatorRegistry.ts

```diff
@@ -0,0 +1,56 @@
+import { Lifecycle, scoped } from 'tsyringe'
+import { TheGraphClient, Logger } from '@streamr/utils'
+import { shuffle } from 'lodash'
+import { NetworkPeerDescriptor } from '../Config'
+import { LoggerFactory } from '../utils/LoggerFactory'
+
+@scoped(Lifecycle.ContainerScoped)
+export class OperatorRegistry {
+    private readonly theGraphClient: TheGraphClient
+    private readonly logger: Logger
+
+    constructor(
+        theGraphClient: TheGraphClient,
+        loggerFactory: LoggerFactory
+    ) {
+        this.theGraphClient = theGraphClient
+        this.logger = loggerFactory.createLogger(module)
+    }
+
+    async findRandomNetworkEntrypoints(
+        maxEntryPoints: number = 5,
+        maxQueryResults: number = 50, 
+        maxHeartbeatAgeHours: number = 24,
```

Comment:
In the future these could be fields of `EntrypointDiscovery` and the defaults defined in `config.schema.json`.